angular/core/components/poll_page/poll_page_controller.coffee

```diff
@@ -1,13 +1,12 @@
 angular.module('loomioApp').controller 'PollPageController', ($scope, $rootScope, $routeParams, MessageChannelService, Records, $location, ModalService, PollService, PollCommonOutcomeModal, PollCommonEditVoteModal, PollCommonShareModal, Session) ->
-  $rootScope.$broadcast('currentComponent', { page: 'pollPage', skipScroll: true })
```

Comment:
I can't help but feel there's some reason for this being pre-model load. Could you ensure that this still works properly for, say, loading a private discussion while logged out?

Reply:
Ok, that test seems to work fine.